plugins/gateway/tyk/deploy/deploy.sh

```diff
@@ -2,12 +2,43 @@
 TykProxyContainerName="${TYK_PROXY_CONTAINER_NAME:-tyk-gtw}"
 TykGatewayDeploymentName="${TYK_GATEWAY_DEPLOYMENT_NAME:-tyk-gtw}"
 TykGatewayDeploymentNamespace="${TYK_GATEWAY_DEPLOYMENT_NAMESPACE:-default}"
-UpstreamTelemetryHostName="${UPSTREAM_TELEMETRY_HOST_NAME:-apiclarity-apiclarity.apiclarity:9000}"
+UpstreamTelemetryHostName="${UPSTREAM_TELEMETRY_HOST_NAME:-apiclarity-apiclarity.apiclarity}"
+UpstreamTelemetryHTTPPort="${UPSTREAM_TELEMETRY_HTTP_PORT:-9000}"
+UpstreamTelemetryTLSPort="${UPSTREAM_TELEMETRY_TLS_PORT:-9443}"
```

Comment:
UPSTREAM_TELEMETRY_HTTP_PORT, UPSTREAM_TELEMETRY_TLS_PORT - 
Should it be part of the env variables and helm options?

Reply:
in general we are not configuring the ports via helm values so no need.